### ISSUE TITLE
500 erros on `api/query` route

### ISSUE THREAD

**Issue by chobberoni** [daily-demos/ai-meeting-assistant#13]
## Feedback
1. After several minutes in the call we are seeing a ton of 500 errors and 504 errors when trying to query the summarizer or the question query box it also references cross origin site issues 
2. The summarizer becomes unusable and does not return anything
3. The query box becomes unusable

**Comment by lazeratops** [comment]
Thanks, @chobberoni. Could you please attach your client-side logs from the session? Additionally, did this happen locally or with the deployed application? If locally, please provide the server logs as well. If deployed, approximately when did you reproduce this? I can try to find the logs in EB.

**Comment by chobberoni** [comment]
> Thanks, @chobberoni. Could you please attach your client-side logs from the session? Additionally, did this happen locally or with the deployed application? If locally, please provide the server logs as well. If deployed, approximately when did you reproduce this? I can try to find the logs in EB.

1. All in the production demo hosted on vercel/AWS
2. Client-side logs -- I dd not save as I was in a real call while this happened. EB-logs will have them -- it will be the most recent session

**Comment by lazeratops** [comment]
The server logs from last week indicate one error: attempting to query a session that does not exist.

After checking the session in question, I confirmed that the bot was ejected as expected when the room expired. The default room expiry is 15 minutes and participants are set to be ejected by default. The bot was ejected as expected at this time. This would mean anyone trying to query this session after this time would fail, because the session no longer exists (as designed).

But the problem is that somehow participants _were_ able to query this session despite it having been expired, and I suspect you did not ping the assistant API directly to do so (which would have explained this), right? In the dashboard, I saw some strange activity in human participants' session events for this session and will start a thread in Slack to follow up, as that is more of an underlying API question.
